css/mobile-home.css

```diff
@@ -13,7 +13,7 @@ p {
 }
```

Comment:
# Optional
- Please adjust the color of the top side of the Mobile size to make it like the top section color. I provided the pictures for you to understand.
- Your Design.

![Capture](https://user-images.githubusercontent.com/63972670/203732274-ff8eceed-a1e4-4a3a-ac1a-4ec19b2e1ead.PNG)
- Original
![Capture1](https://user-images.githubusercontent.com/63972670/203732358-94b36dee-f34f-4855-853a-647bb585a35e.PNG)

Reply:
Thank you, I will be doing that.